deploy/docker-compose.yml

```diff
@@ -7,12 +7,9 @@ services:
     image: prom/prometheus
     command:
       - "--config.file=/config/prometheus.yml"
-      - "--storage.tsdb.path=/prometheus"
-      - "--web.console.libraries=/usr/share/prometheus/console_libraries"
-      - "--web.console.templates=/usr/share/prometheus/consoles"
     ports:
       - "9090:9090"
-    user: "65534:0"
+    user: root
```

Comment:
This might have a better solution, it is because the mounted dir access right it seems, can take a look on that